tests/test_remove_mon_from_cluster.py

```diff
@@ -0,0 +1,82 @@
+"""
+A Testcase to remove mon from
+when I/O's are happening.
+
+Polarion-ID- OCS-355
+
+"""
+
+import logging
+import time
+import pytest
+from ocs_ci.ocs import ocp, constants
+from ocs_ci.framework.testlib import tier4, ManageTest
+from ocs_ci.framework import config
+from ocs_ci.ocs.resources import pod
+from tests.helpers import run_io_with_rados_bench, delete_cephblockpool
+from ocs_ci.ocs.cluster import CephCluster
+
+
+log = logging.getLogger(__name__)
+
+POD = ocp.OCP(
+    kind=constants.POD, namespace=config.ENV_DATA['cluster_namespace']
+)
+
+
+def verify_mon_pod_up():
+    """
+    Verify mon pods are in Running state.
+
+    Returns:
+        bool: True for wait for the resource, False otherwise
+
+    """
+    log.info(f"Verifying all mons pods are up and Running")
+    time.sleep(15)
```

Comment:
Is the sleep necessary? considering that below wait_for_resouece() is called with 700 seconds T/O